python/pyspark/mllib/regression.py

```diff
@@ -37,10 +37,11 @@ class LabeledPoint(object):
     """
     Class that represents the features and labels of a data point.
 
-    :param label: Label for this data point.
-    :param features: Vector of features for this point (NumPy array,
-            list, pyspark.mllib.linalg.SparseVector, or scipy.sparse
-            column matrix)
+    :param label:
+      Label for this data point.
+    :param features:
+      Vector of features for this point (NumPy array, list,
+      pyspark.mllib.linalg.SparseVector, or scipy.sparse column matrix)
```

Comment:
Sorry, my bad - it is already capitalized, my font was too small I guess :).  Could you just add a period after the end of the description?  So should read ..."scipy.sparse column matrix)."